WTraining/test/users/post.spec.js

```diff
@@ -0,0 +1,118 @@
+const request = require('supertest');
+const { factory } = require('factory-girl');
+const app = require('../../app');
+const { factoryByModel } = require('../factory/factory_by_models');
+const { USER_ERROR, DATABASE_ERROR, INTERNAL_ERROR, CREATED } = require('../constants/constants');
+
+const ENDPOINT = '/users';
+
+let response = {};
+let userData = {};
+
+describe(`POST ${ENDPOINT}`, () => {
+  beforeAll(async done => {
+    await factoryByModel('User');
+    userData = await factory.build('User', { password: 'ABC123ab', email: 'asasas@wolox.com.ar' });
+    done();
+  });
+
+  describe('Should handle error when password does not meet requirements', () => {
+    beforeEach(async done => {
+      const user = {
+        name: userData.dataValues.name,
+        last_name: userData.dataValues.lastName,
+        email: userData.dataValues.email,
+        password: 'erre.'
+      };
+
+      response = await request(app)
+        .post(ENDPOINT)
+        .send(user);
+
+      done();
+    });
+
+    test('Status code should be 503', () => {
```

Comment:
I think that error code when password is missing or is invalid could be `400` or` 422` but I suppose that code error is `503` because at moment does not schema validation. Remember this case because it is posible that  you have to changed it